sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -533,8 +533,6 @@ public List<EventParticipantIndexDto> getIndexList(
 				.and(cb, filter, cb.equal(samples.get(Sample.PATHOGEN_TEST_RESULT), eventParticipantCriteria.getPathogenTestResult()));
 		}
 
-		cq.distinct(true);
```

Comment:
What's the reason for removing the distinct clause here?

Reply:
it is not needed, was added by Sergiu trying to fix the problem, so i didn't see a reason why it would be needed.